### ISSUE TITLE
[Bug Report] VTab to and VTabItem combination crashing browser tab

### ISSUE THREAD

**Issue by loonpwn** [vuetifyjs/vuetify#5762]
### Versions and Environment
**Vuetify:** 1.3.11
**Vue:** 2.5.17
**Browsers:** Chrome 70.0.3498.0, Safari, Mozilla Firefox
**OS:** Mac OS 10.13.4

### Steps to reproduce
The exact point of failure is having v-tabs and a v-tab-item and then adding a `to` attribute to one of the `v-tab`s

```
<v-tabs>
  <v-tab to="/test">tab1</v-tab>
  <v-tab>tab2</v-tab>
  <v-tab>tab3</v-tab>

  <v-tab-item></v-tab-item>
</v-tabs>
```

### Expected Behavior
The browser should not crash. I'd like to be able to use both a `to` and the `v-tab-item` so that users can visit tabs via direct link and use the tabs as usual 

### Actual Behavior
The tab freezes and becomes unresponsive. 

### Reproduction Link
<a href="https://codepen.io/anon/pen/QJJLNg" target="_blank">https://codepen.io/anon/pen/QJJLNg</a>

### Other comments
There may be a better way of doing my use case, but the fact it's crashing the browser isn't good.

**Comment by johnleider** [comment]
Workaround is to explicitly bind the value of  `v-tab-item`: https://codepen.io/johnjleider/pen/VVVZVe

**Comment by johnleider** [comment]
Fixed in f22d754
